aggregation/examples/swap-with-permit/main.go

```diff
@@ -33,7 +31,7 @@ const (
 )
 
 func main() {
-	config, err := aggregation.NewDefaultConfiguration(nodeUrl, privateKey, constants.PolygonChainId, "https://api.1inch.dev", devPortalToken)
+	config, err := aggregation.NewConfiguration(nodeUrl, privateKey, constants.PolygonChainId, "https://api.1inch.dev", devPortalToken)
```

Comment:
Are there other URLs they can provide here? Seems like this should be statically used inside of `NewConfiguration`

Reply:
What do you mean?

Reply:
I think we can just remove "https://api.1inch.dev" from the parameters entirely and just use it as a constant somewhere in the SDK

Reply:
I do think that we do not need to hardcode the URL, since it could be different potentially